packages/middleware/src/websandbox/sandbox/sandbox.ts

```diff
@@ -114,6 +114,7 @@ export async function run(params: {
     const content = readFileSync(params.path, 'utf-8');
     const esBuildResult = esbuild.transformSync(content, {
       format: 'cjs',
+      banner: '"use strict";',
```

Comment:
In `vc dev` all middleware ultimately runs through the sandbox, which does not bundle, but instead transforms imported code as it is imported. Therefore, adding `'use strict';` to the top of each imported file is the same as adding the build's final singular `'use strict';` added to the top of the file.